README.md

```diff
@@ -6,9 +6,12 @@
 
 # gulp
 **The streaming build system**
-
 [![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Support us][gittip-image]][gittip-url] [![Build Status][travis-image]][travis-url] [![Coveralls Status][coveralls-image]][coveralls-url] [![Gitter chat][gitter-image]][gitter-url]
 
+## What is gulp?
+
+Gulp is a build system, meaning that you can use it to automate common tasks in the development of a website, it's allows you to compile preprocessed CSS, minifying JavaScript and reloading the browser and much more.
```

Comment:
How about this:

gulp is a toolkit that will help you automate painful or time-consuming tasks in your development workflow. For web development (if that's your thing) it can help you with web development by doing CSS preprocessing, JS transpiling, minification, live reloading, and much more. Integrations are built into all major IDEs and people are loving gulp across PHP, .NET, Node.js, Java, and more. With over 1700 plugins (and plenty you can do without plugins), gulp let's you quit messing with build systems and get back to work.